kb/temporal-platform-limits-sheet.md

```diff
@@ -46,6 +46,7 @@ Warnings are soft limits that produce a warning log on the server side.
     - `limit.numPendingSignals.error`
     - `limit.numPendingCancelRequests.error`
     - `limit.numPendingChildExecutions.error`
+  - By default, [Batch jobs](/cli/batch) are limited to one at a time.
```

Comment:
```suggestion
  - By default, [Batch jobs](/cli/batch) are limited to one at a time.
```
Need an appropriate noun following "one".

Reply:
```suggestion
  - By default, [Batch jobs](/cli/batch) are limited to one job at a time.
```

like this?